src/edu/csus/ecs/pc2/core/PacketHandler.java

```diff
@@ -3536,7 +3536,26 @@ private void checkoutRun(Packet packet, Run run, ClientId whoRequestsRunId, bool
                     // just get run and sent it to them.
 
                     theRun = contest.getRun(run.getElementId());
-                    RunFiles runFiles = contest.getRunFiles(run);
+                    
+                    RunFiles runFiles = null;
+                    
+                    // in case GetRunFiles throws an exception we want to deal with it separately
+                    try {
+                        runFiles = contest.getRunFiles(run);  
+                    } catch (Exception e) {
+                        controller.getLog().info("contest.getRunFiles (R/O) can not get files for run " + run.getNumber() + ": " + e.getMessage());
```

Comment:
I don't think this should be an "info()" log message; the existence of a Run with no RunFiles is clearly an error of some kind.  I would suggest at least type "warning()".